pyproject.toml

```diff
@@ -112,6 +111,7 @@ cryptography = "37.0.1"
 SQLAlchemy = "1.3.24"
 redis = "^4.5.3"
 newrelic-telemetry-sdk = "^0.5.0"
+aiospamc = "0.10"
```

Comment:
this change might require running SL on staging for a couple of days to make sure there's no regression.

Reply:
It's already in staging 👍